### ISSUE TITLE
Add support for uploading SVGs

### ISSUE THREAD

**Issue by LukeTowers** [wintercms/winter#9]
Need to integrate with https://github.com/darylldoyle/svg-sanitizer to automatically sanitize SVGs on upload and then re-add them to the list of allowed defaultExtensions, assetExtensions, and imageExtensions. 

Original october#5261

**Comment by qroft** [comment]
Hi Luke,
i am too noobish to know if what i found out fits into your request for integrating the sanitizer or if this is a bug
that i have encountered.
I have allowed the SVG file type inside the cms.php configuration.
When i try to upload a SVG file through the Media manager A) i do not get an error B) the file is not listed and uploaded.

So as a workaround i thought: let's upload my 6 SVG files manually via FTP.
What happens is A) the files are uploaded B) the Media manager shows me "6 item(s)" to the right of the folder but when i C) enter the folder, no images are shown.

Now when i call the full path of the uploaded SVG, it is shown.
**In short: the SVG is uploaded and can be used, but the Media manager does not show it.**

**Comment by LukeTowers** [comment]
What does your cms.php configuration look like?

**Comment by shadow5og** [comment]
Hello There

Any update on this?

**Comment by bennothommo** [comment]
@shadow5og not at this stage, but you can enable SVG support in the Media Manager now if you wish by configuration change:
https://wintertricks.com/tricks/enable-svg-support-in-media-manager

**Comment by shadow5og** [comment]
Alright Brother!

**Comment by LukeTowers** [comment]
Initial support added in https://github.com/wintercms/storm/pull/61, awaiting further integration with the media library.

**Comment by LukeTowers** [comment]
Implemented in https://github.com/wintercms/winter/commit/8445b84c08c16aed818de26ce40e75d4c182fb9e
